### ISSUE TITLE
Unable to run pip-installed pyarmor

### ISSUE THREAD

**Issue by dlindelof** [dashingsoft/pyarmor#36]
Hello,

I was going to try `pyarmor` on my Mac and ran `pip install pyarmor`, which worked fine. But when I run `pyarmor` this is what I get:

```
$ pyarmor
Traceback (most recent call last):
  File "/usr/local/bin/pyarmor", line 11, in <module>
    sys.exit(main_entry())
  File "/usr/local/lib/python2.7/site-packages/pyarmor/__init__.py", line 7, in main_entry
    from pyarmor.pyarmor import main
  File "/usr/local/lib/python2.7/site-packages/pyarmor/pyarmor.py", line 49, in <module>
    from utils import make_capsule, obfuscate_scripts, make_runtime, \
  File "/usr/local/lib/python2.7/site-packages/pyarmor/utils.py", line 55, in <module>
    search_pytransform(PYARMOR_PATH)
  File "/usr/local/lib/python2.7/site-packages/pyarmor/utils.py", line 49, in search_pytransform
    raise RuntimeError('No library %s found', src)
RuntimeError: ('No library %s found', '/usr/local/lib/python2.7/site-packages/pyarmor/platforms/macosx_x86_64/_pytransform.dylib')
```

My Python was installed through Homebrew.

Any idea what might be the problem?

**Comment by BrendanSimon** [comment]
I got the same error.  However I could get pyarmor to work if use `sudo` first (though I don't think that's the solution)

`sudo /Library/Frameworks/Python.framework/Versions/3.6/bin/pyarmor obfuscate --src=. --entry=myapp.py`

Your workaround of copying the `_pytransform.dylib` did work for me :)

```
cd /Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pyarmor
sudo cp platforms/macosx_x86_64/_pytransform.dylib .
```

BTW, on my Mac with High Sierra installed, `platform.system().lower()` gives me `darwin`

```
Python 3.6.7 (v3.6.7:6ec5cf24b7, Oct 20 2018, 03:02:14) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform
>>> print(platform.system().lower())
darwin
```

**Comment by jondy** [comment]
The platform name issue I have fixed from v4.2.3, now it's `macosx_x86_64`

When run `pyarmor.py` at first time, it will check current platform, and copy the corresponding dynamic library to the same path of `pyarmor.py`. If current user has no privilege to write this directory, `sudo` has to stand out.  After that, no `sudo` should work.

**Comment by 52pingo** [comment]
Hi, I'd like to apply for a personal license.
My license no: pyarmor-vax-000000
Thank you!
